Add blog post about the good first issue board (#45)
vapor/blog: Content/posts/first-issue-board-available.md

---
date: 2022-04-22 15:00
description: Have you thought about contributing to Vapor but you don't know where to start? Check out the good first issue projectboard.
tags: contributing, github
author: Benny
---
# The beginner issues project board is here

So you want to contribute to Vapor but you're a beginner. Or maybe you want to do some open-source contribution but you don't know where to start. Then we may have the solution for you.

## Why have we created this project board?

As new developers or even experienced developers, we've probably all been in a position of wanting to create something or wanting to contribute to a project but not really knowing where to start.  

> That's why not too long ago we've implemented a workflow that takes issues labeled with the `good first issue` label from all repositories in the Vapor project and creates them as cards on the [Beginner Issues projectboard](https://github.com/orgs/vapor/projects/10). These issues are generally not too hard to solve but they're a great gateway into contributing and getting to work on Vapor. So do check it out if you're interested.  

Reviewer: Nit language tweaks

```suggestion
That's why not too long ago we implemented a workflow that takes issues labeled with the `good first issue` label from all repositories in the Vapor organisation and creates them as cards on the [Beginner Issues Project Board](https://github.com/orgs/vapor/projects/10). These issues are generally not too hard to solve but they're a great gateway into contributing to Vapor. So do check the board out if you're interested.  
```
